.gitignore

```diff
@@ -0,0 +1,3 @@
+.cache
+build
+.vscode
```

Comment:
These files should be specified at each user's global git ignore settings (i.e. ".config\git\ignore"). Please remove this .gitignore file.